contracts/DCAStrategies/DCAStrategies/DCAStrategiesPositionsHandler.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.8.7 <0.9.0;
+
+import '../../interfaces/IDCAStrategies.sol';
+
+abstract contract DCAStrategiesPositionsHandler is IDCAStrategiesPositionsHandler {
+  function deposit(
+    uint80 _strategyId,
```

Comment:
We will probably need to take the hub's address here
```suggestion
    IDCAHub _hub,
    uint80 _strategyId,
```

Reply:
I will skip this for now because I'm getting **_stack too deep_** on that function 💀. Maybe we can store the hub in _Strategy_ struct?

Reply:
Damn you solidity!!!

In that case, let's do like Sam [here](https://github.com/Mean-Finance/dca-v2-periphery/blob/main/contracts/interfaces/IDCAHubSwapper.sol#L151), and take a struct as a parameter

Reply:
That's better